node/src/components/deploy_acceptor.rs

```diff
@@ -493,10 +493,7 @@ impl DeployAcceptor {
                     })
             }
             ExecutableDeployItemIdentifier::Package(
-                ref
-                contract_package_identifier
-                @
-                ContractPackageIdentifier::Hash {
+                ref contract_package_identifier @ ContractPackageIdentifier::Hash {
```

Comment:
Since these two codeblocks happen to be the only blockers to using older versions of stable Rust, could you just change them to:
```rust
ExecutableDeployItemIdentifier::Package(ContractPackageIdentifier::Hash {
    contract_package_hash,
    version: maybe_package_version,
}) => {
```
and then remove the line
```rust
let maybe_package_version = contract_package_identifier.version();
```

Reply:
Addressed